src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -1012,6 +1215,45 @@ private void callRestEndpoint(String s2sToken, EventInformation eventInformation
             .statusCode(HttpStatus.NO_CONTENT.value());
     }
 
+    private Response getEventToRestEndpoint(String messageId,
+                                            String s2sToken) {
+        return given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, s2sToken)
+            .when()
+            .get("/messages/" + messageId);
+    }
+
+    private Response postEventToRestEndpoint(String messageId,
+                                             String s2sToken,
+                                             EventInformationRequest eventInformation) {
+        return given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, s2sToken)
+            .body(asJsonString(eventInformation))
+            .when()
+            .post("/messages/" + messageId);
+    }
+
+    private Response putEventToRestEndpoint(String messageId, String s2sToken,
+                                            EventInformationRequest request,
+                                            boolean fromDlq) {
+        return given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, s2sToken)
+            .body(asJsonString(request))
+            .when()
+            .put("/messages/" + messageId + (fromDlq ? "?from_dlq=true" : ""));
+    }
+
+    private Response getMessageToRestEndpoint(String messageId, String s2sToken) {
```

Comment:
Is this same content as `getEventToRestEndpoint` ?

Reply:
Actually yes and getMessageToRestEndpoint is not used. Will remove.